### ISSUE TITLE
web3 v1 upgrade

### ISSUE THREAD

**Issue by rojosnow** [MARKETProtocol/MARKET.js#144]
### Why Is this Needed?

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: Upgrade all applications that depend upon web3 to the latest v1.

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Enhancement

### Current Behavior
[comment]: # (Describe what actually happened.)
Currently, we are on web3 0.20.6.

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
All MARKET applications need to be upgraded to the latest v1.

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
- All applications work on v1 of web3.

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)

**Comment by rojosnow** [comment]
Once this TypeChain issue is resolved, we can start work, https://github.com/krzkaczor/TypeChain/issues/25#issuecomment-413360895

**Comment by perfectmak** [comment]
@rojosnow I believe this is no more blocked, so we can start planning out the epic.

**Comment by rojosnow** [comment]
@perfectmak You are correct! I think this will be a big piece of October's release. Let's chat more on Monday.
